specification/schemas/return-orders/DemoOrderResponse.json

```
@@ -0,0 +1,73 @@
+{
+  "allOf": [
+    {
+      "$ref": "#/components/schemas/DemoOrder"
+    },
+    {
+      "required": [
+        "id",
+        "attributes",
+        "relationships",
+        "meta"
+      ],
+      "properties": {
+        "attributes": {
+          "required": [
+            "created_at",
+            "recipient_address",
+            "items"
+          ],
+          "properties": {
+            "recipient_address": {
+              "required": [
+                "first_name",
+                "last_name",
+                "street_1",
+                "city",
+                "country_code",
+                "email"
+              ]
+            },
+            "items": {
+              "items": {
+                "required": [
+                  "id",
+                  "name",
+                  "description",
+                  "quantity",
+                  "item_price"
+                ]
+              }
+            }
+          }
+        },
+        "relationships": {
+          "required": [
+            "shop"
+          ]
+        },
+        "meta": {
+          "type": "object",
+          "additionalProperties": false,
+          "description": "Properties indicating whether the order was sent with a shipping method that supports returning the shipment with a previously provided label",
+          "required": [
+            "shipping_and_return",
+            "label_in_the_box",
+            "package_type"
+          ],
+          "properties": {
+            "shipping_and_return": {
+              "type": "boolean"
+            },
+            "label_in_the_box": {
+              "type": "boolean"
+            },
+            "package_type": {
+              "$ref": "#/components/schemas/PackageType"
+            }
+          }
+        }
```

These meta properties are not necessary, they actually come from the carrier integration using the outbound shipment identifier. 
```suggestion
        }
```